### ISSUE TITLE
Range component is incorrectly displayed in firefox

### ISSUE THREAD

**Issue by zarofab** [ec-europa/europa-component-library#3377]
### Expected behavior

The track of the input range component should be colored:

![image](https://github.com/ec-europa/europa-component-library/assets/117382087/48bc81f8-a45f-4f94-aa86-4a76bbfd4185)

### Actual behavior

The track is not colored:

![image](https://github.com/ec-europa/europa-component-library/assets/117382087/d7abe91f-f6ed-4c01-a643-3ef6ca6b7a99)

### Steps to reproduce the issue

Open the documentation for the range component in Firefox https://ec.europa.eu/component-library/ec/components/forms/range/code/

To fix, I have added the following snippet in my custom CSS:
```css
.ecl-range::-moz-range-track {
  background-color: var(--c-n);
}
```

**Comment by planctus** [comment]
Thanks for opening this issue, @zarofab, we will fix this as soon as possible

**Comment by emeryro** [comment]
Fixed in 4.2.0
